monai/bundle/config_item.py

```diff
@@ -337,3 +367,19 @@ def is_expression(cls, config: Union[Dict, List, str]) -> bool:
 
         """
         return isinstance(config, str) and config.startswith(cls.prefix)
+
+    @classmethod
+    def is_import_statement(cls, config: Union[Dict, List, str]) -> bool:
+        """
+        Check whether the config is an import statement (a special case of expression).
+
+        Args:
+            config: input config content to check.
+        """
+        if not cls.is_expression(config):
+            return False
+        if "import" not in config:
+            return False
+        return isinstance(
+            first(ast.iter_child_nodes(ast.parse(f"{config[len(cls.prefix) :]}"))), (ast.Import, ast.ImportFrom)
```

Comment:
This line is quite complicated, could you please help add some unit tests to cover all the cases?

Thanks in advance.